src/components/play/PlayUI.js

```diff
@@ -538,25 +597,26 @@ class PlayUI extends Component {
         }
 
         this.scheduleToneEvents()
+        await this.renderPatternPresetsSequencer();
     }
 
     scheduleToneEvents() {
         this.disposeToneEvents()
         const _this = this
         this.toneParts = []
         for (const layer of this.round.layers) {
-            const notes = this.convertStepsToNotes(layer.steps, this.userColors[layer.createdBy])
+            const color = layer.isMuted ? '#FFFFFF' : this.userColors[layer.createdBy]
+            const notes = this.convertStepsToNotes(layer.steps, color)
             for (let note of notes) {
                 note.time += 'i';
             }
             let part = new Tone.Part(function (time, note) {
                 Tone.Draw.schedule(function () {
```

Comment:
Could you use a fat arrow and just use `this`?

Tone.Draw.schedule(() => { this.doAThing }

same for `let part = `

Reply:
I think we can dedicate a branch to getting rid of these. Let's ignore them for now.